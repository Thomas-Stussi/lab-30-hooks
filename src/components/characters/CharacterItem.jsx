import React from 'react';
import PropTypes from 'prop-types';

const CharacterItem = ({ name, image }) => (
  <>
    <img src={image} />
    <h3>{name}</h3>
  </>
);

CharacterItem.propTypes = {
  name: PropTypes.string.isRequired,
  image: PropTypes.string.isRequired
};

export default CharacterItem;
