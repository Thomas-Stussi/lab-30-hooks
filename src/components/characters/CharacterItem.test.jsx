/* eslint-disable max-len */
import React from 'react';
import { render } from '@testing-library/react';
import CharacterItem from './CharacterItem';

describe('Item Component', () => {
  it('renders Item component', () => {
    const { asFragment } = render(<CharacterItem name="test" image="https://www.placecage.com/c/140/200" />);

    expect(asFragment()).toMatchSnapshot();
  });
});
