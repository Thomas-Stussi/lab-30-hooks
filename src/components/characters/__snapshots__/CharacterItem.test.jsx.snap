// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Item Component renders Item component 1`] = `
<DocumentFragment>
  <img
    src="https://www.placecage.com/c/140/200"
  />
  <h3>
    test
  </h3>
</DocumentFragment>
`;
